app/queries/standings_resolver.rb

```diff
@@ -0,0 +1,9 @@
+class StandingsResolver < ApplicationQuery
+  def standings
+    @standings ||= current_school.standings.where(archived_at: nil)
```

Comment:
```suggestion
    @standings ||= current_school.standings.live
```

make it scope? live/active?